superagi/resource_manager/file_manager.py

```diff
@@ -45,7 +56,7 @@ def write_to_s3(self, file_name, final_path):
                 s3_helper = S3Helper()
                 s3_helper.upload_file(img, path=resource.path)
 
-    def write_file(self, file_name: str, content):
+    def write_file(self, file_name: str, content, return_file_path: bool = False):
         if self.agent_id is not None:
```

Comment:
Dont pass return_file_path, here in write file path param

Reply:
It was aiding another function, seems like obsolete now, removing it!